src/pdf/ShippingLabels/ShippingLabelsPdf.tsx

```diff
@@ -5,7 +5,7 @@ import { Document, Page, Text, View, StyleSheet } from "@react-pdf/renderer";
 
 export interface ParcelClients {
     packing_datetime: string;
-    collection_centre: string;
+    collection_centre: string ;
```

Comment:
Weird that ES lint didn't fix the space...

Reply:
Interesting it wasn't in my local computer - thanks for spotting it